packages/account/src/providers/provider.ts

```diff
@@ -612,13 +612,26 @@ export default class Provider {
    * @returns A promise that resolves to the Chain and NodeInfo.
    */
   async fetchChainAndNodeInfo() {
-    const nodeInfo = await this.fetchNode();
-    Provider.ensureClientVersionIsSupported(nodeInfo);
-    const chain = await this.fetchChain();
+    const { nodeInfo, chain } = await this.operations.getChainAndNodeInfo();
+
+    const processedNodeInfo: NodeInfo = {
+      maxDepth: bn(nodeInfo.maxDepth),
+      maxTx: bn(nodeInfo.maxTx),
+      nodeVersion: nodeInfo.nodeVersion,
+      utxoValidation: nodeInfo.utxoValidation,
+      vmBacktrace: nodeInfo.vmBacktrace,
+    };
+
+    Provider.ensureClientVersionIsSupported(processedNodeInfo);
```

Comment:
@nedsalk We still print the warning tho?

https://github.com/FuelLabs/fuels-ts/pull/3284/files#r1793066249

Reply:
Yes.